sygnal/notifications.py

```diff
@@ -167,7 +168,7 @@ def __init__(self, name: str, sygnal: "Sygnal", config: Dict[str, Any]):
 
     async def dispatch_notification(
         self, n: Notification, device: Device, context: "NotificationContext"
-    ) -> List[str]:
```

Comment:
why are these annotations going away? :sob: 
I think they probably want to come back :)